examples/resources/awscc_rds_db_cluster/rds_db_cluster_create.tf

```diff
@@ -0,0 +1,7 @@
+resource "awscc_rds_db_cluster" "example_db_cluster" {
+  availability_zones    = ["us-east-1b", "us-east-1c"]
+  engine                = "aurora-mysql"
+  db_cluster_identifier = "example-dbcluster"
+  master_username       = "foo"
+  master_user_password  = "foobarbaz"
+}
```

Comment:
Great example, is it also good to also expand one more example for handling the password using managed master password via Secrets Manager. I think most customers would benefit from this if they don't want to put the plaint text password in the Git version control.
e.g. [ manage_master_user_password ] or [ master_user_secret ]

Reply:
I've updated this example to used the manage_master_user_password = true parameter to better align to best practices as well as updating the corresponding docs

Reply:
LGTM,thanks